assets/typst/client/game_ui.typ

```diff
@@ -192,96 +206,126 @@
 
 // Rule
 // - Each box represents 10 HP
-#let playerhealth(current_hp, max_hp, rect_width: 21pt, rect_height: 31pt, spacing: 8pt) = {
-    // Calculate the ratio of current HP to max HP
-    let hp_ratio = current_hp / max_hp
+#let playerhealth(
+  current_hp,
+  max_hp,
+  total_width: 280pt,
+  rect_height: 31pt,
+  spacing: 8pt,
+) = {
+  // Calculate the total number of blocks needed
+  let blocks = calc.ceil(current_hp / 10)
+  let max_blocks = calc.ceil(max_hp / 10)
```

Comment:
Wut do you think? `ceil` or `floor` is better? I am ok with both, cuz going with what's better.

Reply:
I think ceil is better to avoids situations where players might feel penalized because the next box is missing until they hit the next full 10 and also if floor is used, there could be cases where the health display doesn't visually match the actual HP (examples: 9 HP showing 0 blocks).